webapp/src/pages/FoodCheckout.js

```diff
@@ -95,33 +96,51 @@ export default function FoodCheckout() {
       });
   }
 
+  const canPlace =
+    checkout.fulfillmentOptionId &&
+    (chosenInstrument || !checkout.requiresPaymentInstrument);
   return (
-    <Row>
-      <LinearBreadcrumbs back={`/cart/${checkout.offering.id}`} />
-      <CheckoutPayment
-        checkout={checkout}
-        selectedInstrument={chosenInstrument}
-        onSelectedInstrumentChange={(pi) => setManuallySelectedInstrument(pi)}
-        onCheckoutChange={(attrs) =>
-          setCheckoutMutations({ ...checkoutMutations, ...attrs })
-        }
-      />
-      <hr />
-      <CheckoutFulfillment
-        checkout={checkout}
-        onCheckoutChange={(attrs) =>
-          setCheckoutMutations({ ...checkoutMutations, ...attrs })
-        }
-      />
+    <>
+      <LayoutContainer gutters>
+        <LinearBreadcrumbs back={`/cart/${checkout.offering.id}`} />
+      </LayoutContainer>
+      {!canPlace && (
```

Comment:
I think you want `checkout.requiresPaymentInstrument` here? Otherwise if they have an instrument, they won't be able to choose a different one.